doc/source/ray-metrics.rst

```diff
@@ -0,0 +1,131 @@
+Monitoring Ray
+==============
+To help monitoring Ray applications, Ray
+
+- Collects Ray's pre-selected system level metrics.
+- Exposes metrics through Prometheus endpoints.
+- Supports application-level metrics collection APIs.
+
+.. note::
+
+    It is currently an experimental feature and under active development. APIs are subject to change.
+
+Getting Started (Single Node)
+-----------------------------
+You can access Ray's metrics through Prometheus endpoints.
+
+First, let's learn how to access metrics when you use `ray.init()`.
+
+.. code-block:: python
+
+    import ray
+    ray.init(_metrics_export_port=8080)
+    import time
+    time.sleep(100)
+
+Now you can access Prometheus endpoint at the address `http://localhost:8080`. Note that if you don't specify `_metrics_export_port`, 
+the port is randomly assigned. You can check the port number when you run `ray.init`.
```

Comment:
Should we do the same thing we do with the dashboard? This seems like a bad UX

Reply:
I think it is fine because users will eventually just need to run Prometheus with json files for service discovery. I'd love to change it if there's any user request in the future though.